src/xo-app/home/host-item.js

```diff
@@ -115,11 +116,11 @@ export default class HostItem extends Component {
               &nbsp;&nbsp;
               <Tooltip
                 content={isEmpty(host.current_operations)
-                  ? _(`powerState${host.power_state}`)
-                  : <div>{_(`powerState${host.power_state}`)}{' ('}{map(host.current_operations)[0]}{')'}</div>
+                  ? toolTipContent
+                  : <div>{toolTipContent}{' ('}{map(host.current_operations)[0]}{')'}</div>
                 }
               >
-                {isEmpty(host.current_operations)
+                {isEmpty(host.current_operations) && (host.power_state !== 'Running' || host.enabled)
```

Comment:
If I understand correctly, before the handling of "disabled" state, the host icon could have been orange IF there was `current_operations` not empty. Because we added the disabled thing, we must take this case into account.